docs/src/main/sphinx/release/release-449.md

```diff
@@ -2,6 +2,7 @@
 
 ## General
 
+* Add [OpenLineage event listener plugin](/admin/event-listeners-openlineage). ({issue}`21265`)
 * Add an event listener which exposes collected events to an HTTP endpoint. ({issue}`22158`)
```

Comment:
What is this? I think this is wrong and a cut and paste error @colebow  .. right .. the HTTP event listener is old - https://trino.io/docs/current/admin/event-listeners-http.html

Reply:
Oh .. it is separate ... but definitely needs docs as well